test/test_other.py

```diff
@@ -3346,6 +3357,54 @@ def test_embind_jsgen_method_pointer_stability(self):
     # AOT JS generation still works correctly.
     self.do_runf('other/embind_jsgen_method_pointer_stability.cpp', 'done')
 
+  def build_dylink_lib(self, filename, outfile='liblib.so', emcc_args=None):
+    self.clear_setting('MAIN_MODULE')
+    self.set_setting('SIDE_MODULE')
+    cmd = [compiler_for(filename), filename, '-o', outfile] + self.get_emcc_args()
+    if emcc_args:
+      cmd += emcc_args
+    self.run_process(cmd)
+
+  def prep_dylink_main(self):
+    self.clear_setting('SIDE_MODULE')
+    self.set_setting('MAIN_MODULE', 1)
+
+  @requires_dylink
+  def test_embind_dylink_visibility_hidden(self):
```

Comment:
This looks similar to what `test_dylink_rtti` is doing in `test_core.py`, but with some embind specifics.  As such, perhaps it belongs alongside that test?

Rather than duplicating the three helper functions from test_core.py why not just put this test in test_core.py itself?

Reply:
It also might make more sense alongside `test_embind_no_rtti`

Reply:
Yes, tst_core could make sense as well (I had it there first). Is it primarily a dylink or embind test? test_embind_no_rtti sounds related, I'll check what that does.

(Note I have vacation time is coming up, so I'll get back to this in a couple of week's time.)